tests/integration/requirements.txt

```diff
@@ -0,0 +1,5 @@
+juju>=2,<3
```

Comment:
I'd pin juju to 3.0.4 considering the issues we've encountered

Reply:
I'm going to use juju >3 when converting it to sidecar